frontend/packages/topology/src/components/quick-search/utils/quick-search-utils.tsx

```diff
@@ -53,3 +54,12 @@ export const useTransformedQuickStarts = (quickStarts: QuickStart[]): CatalogIte
     [t, quickStarts, setActiveQuickStart],
   );
 };
+
+export const handleCta = (e: React.SyntheticEvent, item: CatalogItem, clodeModal: () => void) => {
+  e.preventDefault();
+  const { href, callback } = item.cta;
+  if (callback) {
+    clodeModal();
+    removeQueryArgument('catalogSearch');
+  } else history.push(href);
```

Comment:
What am I missing? Looks to me like `callback` is never called.

Reply:
added callback